### PR TITLE
mono-libgdiplus 6.1

### PR DESCRIPTION
**Pull request by EricFromCanada**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/mono-libgdiplus.rb [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
-  url "https://github.com/mono/libgdiplus/archive/6.0.5.tar.gz"
-  sha256 "1fd034f4b636214cc24e94c563cd10b3f3444d9f0660927b60e63fd4131d97fa"
+  url "https://download.mono-project.com/sources/libgdiplus/libgdiplus-6.1.tar.gz"
+  sha256 "97d5a83d6d6d8f96c27fb7626f4ae11d3b38bc88a1726b4466aeb91451f3255b"
   license "MIT"
-  revision 1
+
+  livecheck do
+    url "https://github.com/mono/libgdiplus.git"
+  end
```
Comment (chenrui333) [inline Formula/mono-libgdiplus.rb:10]: cc @samford

**Review by chenrui333 — DISMISSED** [review]

File: Formula/mono-libgdiplus.rb (hunk 1 above)
Comment (samford) [reply Formula/mono-libgdiplus.rb:10]:
```suggestion
  livecheck do
    url "https://download.mono-project.com/sources/libgdiplus/"
    regex(/href=.*?libgdiplus[._-]v?(\d+(?:\.\d+)+)\.t/i)
  end
```

There's something to be said for checking the directory listing page where the `stable` archive is found, as it serves as the download page for the related source files and aligns with the `stable` source. With checking the Git repository, there's a possibility that upstream may tag a new version before the related source file is published on the first-party site and that could cause problems.

File: Formula/mono-libgdiplus.rb [hunk 2]
```diff
@@ -31,15 +35,6 @@ class MonoLibgdiplus < Formula
...
-  # Remove at next version bump.
-  # Upstream PR: https://github.com/mono/libgdiplus/pull/605.
-  # Without this patch, it requires pango 1.43 or lower (current available version is 1.48).
-  patch do
-    url "https://github.com/mono/libgdiplus/commit/8f42e17e92c562cc243844b8a004cd03144b1384.patch?full_index=1"
-    sha256 "b38823891ea201588c1edf29f931a0d353a155d7fac36f114482bbe608c5a1c9"
-  end
 
   def install
     system "autoreconf", "-fiv"
```
Comment (SMillerDev) [inline Formula/mono-libgdiplus.rb:40]: Does it still need this?

Comment (EricFromCanada) [reply Formula/mono-libgdiplus.rb]: Apparently not!

Comment (EricFromCanada) [reply Formula/mono-libgdiplus.rb]: Never mind; turns out that triggers the `flat_namespace` issue.

Comment (carlocab) [reply Formula/mono-libgdiplus.rb]: https://raw.githubusercontent.com/Homebrew/formula-patches/03cf8088210822aa2c1ab544ed58ea04c897d9c4/libtool/configure-big_sur.diff should work

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**